prime-router/settings/prod/0043-hhsprotect.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: "hhsprotect"
```

Comment:
I think it would have been better to do this change in a separate PR.  That way you are not waiting on the rest of the code changes to get it approved.

Reply:
I also think changes to configuration should be tracked by issues, that way we can see the history.

Reply:
The database tracks the history and is the source of truth for these.